app/templates/components/forms/orders/order-form.hbs

```diff
@@ -62,7 +62,7 @@
         {{#each this.allFields.attendee as |field|}}
           {{#if field.isIncluded}}
             {{#if (or field.isFixed this.event.isTicketFormEnabled)}}
-              <form class="field">
+              <form class="field" onsubmit="return false">
```

Comment:
This is a workaround, use preventDefault

Reply:
preventDefault=true